### PR TITLE
Quick idea on a splash screen that will not increase load time

### PR DESCRIPTION
**Pull request by deeptailor**
This is a quick proto of us including a splash screen so users don't see css styles loading in-front of their eyes.

It adds almost no load time.

@charlesh88 @akhenry thoughts?

### CODE REVIEW

**Comment by charlesh88** [comment]
We really need this, the app has been looking very bad while loading. 

I've pushed some changes:

File: src/styles/vue-styles.scss [hunk 1]
```diff
@@ -46,3 +46,7 @@
 @import "../ui/toolbar/components/toolbar-checkbox.scss";
 @import "./notebook.scss";
 @import "../plugins/notebook/components/sidebar.scss";
+
+#splash-screen {
+     display: none;
+}
```
Comment (deeptailor) [inline src/styles/vue-styles.scss:52]: can we add this to the bottom of the file? so we can be sure its loaded fully

**Review by deeptailor — COMMENTED** [review]
Removing comment, disregard

**Comment by deeptailor** [comment]
@akhenry - Just needs your approval (or rejection 😄 )

**Comment by deeptailor** [comment]
@akhenry - Thoughts on this?

**Review by akhenry — APPROVED** [review]
This looks good to me. I do notice that glyphs don't load until after the splash screen has been hidden, but I guess that's because they are resolved asynchronously, and only after the stylesheets have loaded.

**Review by akhenry — CHANGES_REQUESTED** [review]
Just needs a link to an issue.

**Comment by deeptailor** [comment]
Closes #3502

**Comment by deeptailor** [comment]
@akhenry Added an issue

**Review by akhenry — APPROVED** [review]

**Comment by akhenry** [comment]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A - small non-code change
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**